website/content/plugins/drivers/community/wasmtime.mdx

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+page_title: 'Drivers: wasmtime'
+description: >-
+  The Wasmtime task driver is used to run WASI(https://wasi.dev)-compliant binaries.
```

Comment:
I'm not sure where this description actually shows up, but let's not put URLs in there just in case:

```suggestion
  The Wasmtime task driver is used to run WASI-compliant binaries.
```